spec/checkPullRequestLabelsSpec.js

```diff
@@ -537,6 +540,86 @@ describe('Pull Request Label Check', () => {
     });
   });
 
+  describe('when stale build label gets removed before updating branch', () => {
```

Comment:
You need to mock `github.repos.getCommit`. This is what's making the tests fail.

Reply:
working on it.